packages/oui-message/src/index.spec.js

```diff
@@ -1,6 +1,6 @@
 import { filter } from "lodash";
 
-describe("oui-message: ", () => {
+describe("ouiMessage: ", () => {
```

Comment:
Almost perfect, `ouiMessage`, not `ouiMessage: ` :)

Reply:
Indeed. Done! :-)